usecase/parser/utils/parsermanager.go

```diff
@@ -0,0 +1,80 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/crypto-com/chain-indexing/entity/command"
+	applogger "github.com/crypto-com/chain-indexing/internal/logger"
+	"github.com/crypto-com/chain-indexing/usecase/event"
+	"github.com/crypto-com/chain-indexing/usecase/model"
+)
+
+type CosmosParserManager struct {
+	store  map[string]CosmosParserBlockHeight
```

Comment:
I have a small suggestion but now sure if it makes sense.

Shall we define
```
type ParserMessageName string
type ParserBlockHeight string

type CosmosParserManager struct {
	store  map[ParserMessageName]BlockHeightToCosmosParserMap
        ...
}

type BlockHeightToCosmosParserMap map[ParserBlockHeight]CosmosParser
```

I usually find the map key not self-explanatory and affect the readability.

Reply:
yup, that would be more descriptive